src/test/java/tests/BaseClass.java

```diff
@@ -0,0 +1,62 @@
+package tests;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+public class BaseClass {
+    protected WebDriver driver;
+    protected int WAIT_TIMEOUT_SECONDS;
+    Logger logger = Logger.getGlobal();
+
+    protected void waitForPageLoad(WebDriver driver) {
```

Comment:
this method need to be removed from here, you declared same in your Abstract class

Reply:
Refactored, thank you.
Please see in the following commit.